src/cson.coffee

```diff
@@ -60,6 +60,7 @@ parseContents = (objectPath, cachePath, contents, callback) ->
   try
     object = parseObject(objectPath, contents)
   catch parseError
+    parseError.message = "#{objectPath}: #{parseError.message}"
```

Comment:
I'm good with setting error.path but would prefer it to also be in the message / stack trace. Coffeescript sets error.filename but then also provides a custom toString / stack function in order to make it appear.